### DISCUSSION TITLE
PYNQ board file list is empty

### DISCUSSION THREAD

**Discussion by vic9112** [bol-edu/HLS-SOC-Discussions#179] [Q&A]
當我租借pynq後打開卻發現裡面本該存在的資料夾都不在，我想請問是否會影響到驗證的過程，因為我試過租借其他板子，幾乎所有板子都是空的。

![螢幕擷取畫面 2023-12-10 125457](https://github.com/bol-edu/HLS-SOC-Discussions/assets/137171415/6fbd22ad-d50a-4a5e-9710-83b0c235405a)

**Comment by lkl110137918218** [comment]
本來就會是空的，按upload，把需要的檔案上傳

從我的iPhone傳送
